scanapp/forms.py

```diff
@@ -30,3 +30,11 @@ class URLScanForm(forms.Form):
 
 class LocalScanForm(forms.Form):
     upload_from_local = forms.FileField(label='Upload from Local')
+
+
+class UrlSpecialScanForm(forms.Form):
```

Comment:
@pombredanne  I had a discussion with @singh1114  where we planned to have a new view for the URL Scan and keep the URLSpecialScan intact for testing purposes which we would be removing after the main view is created.